packages/php-wasm/compile/build-assets/php_wasm.c

```diff
@@ -4,6 +4,8 @@
  * This file abstracts the entire PHP API with the minimal set
  * of functions required to run PHP code from JavaScript.
  */
+#define off64_t off_t
```

Comment:
@adamziel This one line replaces those off64_t patches.

Reply:
Hm `libphp.a` is built before building and linking `php_wasm.c`. Does defining the macro here actually do the trick?

Reply:
Let me dig a little deeper, although I do believe symbols are preserved in .a files.

Reply:
@adamziel This was driving me insane but I finally realized what was going on here.

musl-**1.2.4** is the version that drops the LFS64 macros:
https://musl.libc.org/releases.html

We're now using emsdk 3.1.43, and version 3.1.4**4** is the version that updates musl to 1.24:

https://github.com/emscripten-core/emscripten/blob/main/ChangeLog.md#3144---072523